File: CS/WinWebSolution.Module.Web/CustomFieldMappingsSchedulerListViewController.cs
using System;
using DevExpress.XtraScheduler;
using System.Collections.Generic;
using DevExpress.Web.ASPxScheduler;

namespace WinWebSolution.Module.Web {
    public class CustomFieldMappingsSchedulerListViewController : CustomFieldMappingsSchedulerListViewControllerBase {
        protected override void AddCustomFieldMappings(List<MappingBase> mappings) {
            ASPxScheduler scheduler = (ASPxScheduler)SchedulerListEditor.Scheduler;
            scheduler.Storage.Appointments.CustomFieldMappings.AddRange(mappings);
        }
    }
}

File: VB/WinWebSolution.Module.Web/CustomFieldMappingsSchedulerListViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraScheduler
Imports System.Collections.Generic
Imports DevExpress.Web.ASPxScheduler
Imports DevExpress.ExpressApp.Scheduler.Web

Namespace WinWebSolution.Module.Web
	Public Class CustomFieldMappingsSchedulerListViewController
		Inherits CustomFieldMappingsSchedulerListViewControllerBase
		Protected Overrides Sub AddCustomFieldMappings(ByVal mappings As List(Of MappingBase))
			Dim scheduler As ASPxScheduler = (CType(Me.SchedulerListEditor, ASPxSchedulerListEditor)).SchedulerControl
			scheduler.Storage.Appointments.CustomFieldMappings.AddRange(mappings)
		End Sub
	End Class
End Namespace
